sip/execution_framework_interface/docker_api/client/test/test_docker_client.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+"""Test for docker client API"""
+
+import logging
+import time
+
+from ..docker_client import DockerClient
+
+logging.basicConfig(level='DEBUG')
+
+DC = DockerClient()
+
+
+def test_create_services():
+    """ Test function for create services"""
+
+    # Remove all services and volumes
+    DC.delete_all_services()
+    assert not DC.get_service_list()
+    time.sleep(2)
```

Comment:
Is this and the other sleeps really needed? Arbitrary length sleeps are never ideal and it adds quite a lot of time to the unit tests.